### DISCUSSION TITLE
Gephi Wishlist: export network into 3D model to 3D print it

### DISCUSSION THREAD

**Discussion by evlacho** [gephi/gephi#2368] [Q&A] [answered]
I wonder if it is possible to export a layout network (e.g., like a Force Atlas 2 network) into a 3D model, which could be subsequently 3D printed

**Comment by wassfila** [comment, marked as answer]
Gephi can export svg, which can be imported by tools such as Fusion360, there you could extrude the shapes you want then export to stl for a slicer tool that would produce gcode.
The edges are lines though, so without thickness.

**Comment by evlacho** [comment]
OK, this sounds interesting. Although I am looking for a real Force Atlas 3D plugin, which then would allow to export and print the network. If I remember correct it used to be one such plugin in earlier versions
